source/onboard/connected-workspaces.rst

```diff
@@ -0,0 +1,284 @@
+
+    In the System Console, system admins can change the **Organization Name**, the **Destination Team**, or channels shared with a remote Mattermost instance as well as channels shared with your local Mattermost instance.
+    
+    1. Under **Connected Workspaces**, identify the connected workspace you want to change.
+    2. Select the **More** |more-icon| icon to the right of the connected workspace, and then select **Edit**.
+
+.. tab:: Slash Commands
+
+    Run the following slash command to remove all secure connections from the current channel:
+
+    ``/share-channel unshare``
+
+    This slash command removes all secure connections from the current channel. A System message notifies you that the channel is no longer shared. Secure connections may continue to be invited to other shared channels.
+
+    Unsharing a shared channel stops synchronizing the channel with the remote Mattermost server; however, the channel continues to function for local users as expected.
+
+Delete a connected workspace
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tab:: System Console
+
+    Deleting a connected server severs the trust relationship between the local Mattermost server and the remote Mattermost server.
+
+    1. Under **Connected Workspaces**, identify the connected workspace you want to remove.
+    2. Select the **More** |more-icon| icon to the right of the connected workspace, and then select **Delete**.
+
+.. tab:: Slash Commands
+
+    Using slash commands, you can uninvite or delete a secure connection from your Mattermost instance.
+    
+    Run the following slash command to uninvite a secure connection:
+
+    ``/share-channel uninvite --connectionID``
+
+    This slash command removes a secure connection from the current channel based on its ``connectionID``. The channel continues to function for local users as expected, and the secure connection may continue to be invited to other shared channels.
+    
+    Run the following slash command to delete a secure connection:
+
+    ``/secure-connection remove --connectionID``
+
+    For example:
+
+    ``/secure-connection remove --connectionID``
+
+    This slash command severs the trust relationship between the local Mattermost server and a remote Mattermost server based on its ``connectionID`` and removes the secure connection from all shared Mattermost channels.
+
+Review connection status
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tab:: System Console
+
+    Under **Connected Workspaces**, you can review all connected workspaces and their current status as one of: **Connected**, **Offline**, or **Connection Pending**.
+
+.. tab:: Slash Commands
+
+    Run the following slash command to review the current status of all secure connections established for your Mattermost instance:
+
+    ``/secure-connection status``
+
+    Status details include:
+
+    - Connection ID
+    - Connection URL
+    - Description
+    - Invite accepted (Yes/No)
+    - Online (Yes/No)
+    - Last ping timestamp (UTC)
```

Comment:
As part of the last batch of changes we added the `Deleted` column to the output of this command

```suggestion
    - Last ping timestamp (UTC)
    - Deleted
```